std_msgs/Header header
uint8 channel
float32 raw_adc
float32 voltage_v
float32 current_ma
float32 percent
float32 engineering_value
